docker-amundsen-atlas.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+services:
+  atlas:
+      image: wbaa/rokku-dev-apache-atlas:latest
+      container_name: atlas_amundsen
+      ports:
+          - 21000:21000
+      networks:
+        - amundsennet
+      environment:
+        - ATLAS_PROVISION_EXAMPLES=true
```

Comment:
that's cool. So Atlas automatically provision example data :)

Reply:
Yes no extra work required ;-)